eox_core/api/v1/urls.py

```diff
@@ -18,3 +20,11 @@
     urlpatterns += [
         url(r'^update-user/$', views.EdxappUserUpdater.as_view(), name='edxapp-user-updater'),
     ]
+
+urlpatterns += [
+    url(
```

Comment:
Is there a specific reason why this URL should be appended to urlpatterns here? If not I think it could be added where urlpatterns is defined, maybe leaving a blank line between this api-docs URL and the rest.

Reply:
I was thinking of having them in a separate set in case we want to disable the view with a setting or something like that.

Reply:
In that case, we could do something similar to what was done with "EOX_CORE_ENABLE_UPDATE_USERS", in the meantime, in my opinion, we should add it where urlpatterns is defined

Reply:
Done.